src/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/TOV.hpp

```diff
@@ -0,0 +1,67 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <memory>
+
+#include "DataStructures/DataVector.hpp"
+#include "NumericalAlgorithms/Interpolation/BarycentricRational.hpp"
+#include "PointwiseFunctions/EquationsOfState/EquationOfState.hpp"  // IWYU pragma: keep
+
+namespace PUP {
+class er;
+} // namespace PUP
+
+// IWYU pragma: no_forward_declare EquationsOfState::EquationOfState
+
+/// Contains functions for TOV solver
+namespace tov {
+
+/*!
+ * \ingroup EinsteinSolutionsGroup
+ * \brief TOV solver based on Lindblom's method
+ *
+ * Uses Lindblom's method of integrating the TOV equations from
+ * <a href="https://arxiv.org/abs/gr-qc/9802072">
+ * "Phase Transitions and the Mass-Radius Curves of Relativistic Stars" </a>
+ * L. Lindblom, Phys.Rev. D58 (1998) 024008
+ *
+ * Instead of integrating \f$m(r)\f$ and \f$p(r)\f$
+ * (\f$r\f$=radius, \f$m\f$=mass, \f$p\f$=pressure)
+ * Lindblom introduces the variables \f$u\f$ and \f$v\f$, with \f$u=r^{2}\f$ and
+ * \f$v=m/r\f$.
+ * The integration is then done with the log of the specific enthalpy
+ * (\f$\mathrm{log}(h)\f$) as the independent variable.
+ *
+ * Lindblom's paper simply labels the independent variable as \f$h\f$.
+ * The \f$h\f$ in Lindblom's paper is NOT the specific enthalpy.
+ * Rather, Lindblom's \f$h\f$ is in fact \f$\mathrm{log}(h)\f$.
+ */
+class InterpolationOutput {
```

Comment:
I'm not a fan of the class name. Maybe think of something more like `TovSolution`